### PR TITLE
Alert-signup updates

### PR DESCRIPTION
**Pull request by chasulin**
- Edits styling of inputs
- Replaces preview graphic with svg that resembles email alert

### CODE REVIEW

File: src/assets/theme/overrides/MuiAutocomplete.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default {
+  root: {
+    backgroundColor: 'white',
+    'border-radius': '4px',
+  },
+};
```
Comment (pnavarrc) [inline src/assets/theme/overrides/MuiAutocomplete.tsx:6]: I'm not sure if this was intended, but the autocomplete box and the email input now have a black border on hover

Kapture 2020-07-20 at 12 00 32

File: src/components/Newsletter/Newsletter.js [hunk 2]
```diff
@@ -211,6 +201,21 @@ class Newsletter extends React.Component {
...
+              id="wurhhh"
+              name="cm-ol-wurhhh"
+              onChange={() => this.setState({ checked: !this.state.checked })}
+              checked={this.state.checked}
+            />
+            <label htmlFor="checkbox">
+              {' '}
+              Also send me <b>daily news</b> with the latest data and scientific
+              findings on COVID{' '}
+            </label>
```
Comment (pnavarrc) [inline src/components/Newsletter/Newsletter.js:217]: nit: why do we have spaces before and after? we can keep them if that makes spacing easier :)

File: src/components/Newsletter/Newsletter.style.tsx [hunk 3]
```diff
@@ -30,14 +34,25 @@ export const StyledNewsletter = styled.div`
...
 
+      ::placeholder {
+        color: #828282;
+        font-size: 15px;
+      }
+
+      &:hover {
+        border: 1px solid black;
+        border-right: none;
+      }
```
Comment (pnavarrc) [inline src/components/Newsletter/Newsletter.style.tsx:54]: oh I see that it was on purpose, never mind

File: src/components/ShareBlock/ShareBlock.tsx [hunk 4]
```diff
@@ -86,7 +86,7 @@ const ShareBlock = ({
           </NewsletterMockupWrapper>
           <NewsletterTextArea>
             <ShareInstructionHeader>Email Alerts</ShareInstructionHeader>
-            <ShareInstructionBody>
+            <ShareInstructionBody alertsInstructions={true}>
```
Comment (pnavarrc) [inline src/components/ShareBlock/ShareBlock.tsx:89]: nit: when the value is `true`, you can just put the name of the property

```jsx
<ShareInstructionBody alertsInstructions>
  {/* ... */}
</ShareInstructionBody>
```

File: src/components/Newsletter/Newsletter.style.tsx (hunk 3 above)
Comment (chasulin) [reply src/components/Newsletter/Newsletter.style.tsx:54]: yeah, this black hover is the default state on the material ui autocomplete component and i spent way tooooo much time trying to figure out how to get rid of it. eventually threw in the towel and just made the email input match

File: src/components/Newsletter/Newsletter.js (hunk 2 above)
Comment (chasulin) [reply src/components/Newsletter/Newsletter.js:217]: ooo im not sure. those were already there. ill nix them

File: src/assets/theme/overrides/MuiAutocomplete.tsx (hunk 1 above)
Comment (chasulin) [reply src/assets/theme/overrides/MuiAutocomplete.tsx:6]: yeah, this black hover is the default state on the material ui autocomplete component and i spent way tooooo much time trying to figure out how to get rid of it. eventually threw in the towel and just made the email input match

**Pull request merged**